src/bin/pgcopydb/summary.c

```diff
@@ -34,13 +34,23 @@ write_table_summary(CopyTableSummary *summary, char *filename)
 	JSON_Value *js = json_value_init_object();
 	JSON_Object *jsObj = json_value_get_object(js);
 
+	char bytesPretty[BUFSIZE] = { 0 };
+	pretty_print_bytes(bytesPretty, BUFSIZE, summary->bytesTransmitted);
+
+	char transmitRate[BUFSIZE] = { 0 };
+	pretty_print_bytes_per_second(transmitRate, BUFSIZE, summary->bytesTransmitted,
+								  summary->durationMs);
+
```

Comment:
I did not want to store bytesPretty and transmitRate in CopyTableSummary structs.

1. It is inexpensive to have an integer addition in our copy loop, unlike pretty printing said value. I did not want to store bytes and bytesPretty when I can only update the primitive type and not keep the other in sync.
2. Calculating the transmitRate in real time is also an expensive operation similar to bytesPretty that was discussed in earlier item. Also, I needed to have the duration information that is not always there when the command is running.
3. If resuming an earlier operation, reading an integer value for bytes from disk makes sense. All the other values can be generated from bytes and duration information.